zebra/zebra_rib.c

```diff
@@ -365,13 +365,15 @@ nexthop_ipv4_ifindex_add (struct rib *rib, struct in_addr *ipv4,
 
 #ifdef HAVE_IPV6
 struct nexthop *
-nexthop_ipv6_add (struct rib *rib, struct in6_addr *ipv6)
+nexthop_ipv6_add (struct rib *rib, struct in6_addr *ipv6, struct in6_addr *src)
 {
   struct nexthop *nexthop;
 
   nexthop = XCALLOC (MTYPE_NEXTHOP, sizeof (struct nexthop));
   nexthop->type = NEXTHOP_TYPE_IPV6;
   nexthop->gate.ipv6 = *ipv6;
+  if (src)
```

Comment:
In this case the following line will cause a crash
https://github.com/Azure/sonic-quagga/blob/pavelsh/src/zebra/zebra_rib.c#L2963